bindings/java/c/host.c

```diff
@@ -276,8 +272,6 @@ static size_t copy_code_fn(struct evmc_host_context* context,
             memcpy(buffer_data, code + code_offset, length);
     }
 
-    (*jenv)->ReleaseByteArrayElements(jenv, jaddress, (jbyte*)address, 0);
```

Comment:
> ReleaseByteArrayElements was called on a pointer that is not derived from GetByteArrayElements causing segfaults

These helpers rely on `CopyDataToJava` which uses `NewByteArray/SetByteArrayRegion`. Are you saying those cannot be freed with it?

Reply:
Can't say for sure, but isn't `NewByteArray` creating a local ref that will be garbage collected? https://docs.oracle.com/javase/10/docs/specs/jni/functions.html#local-references